PowerEditor/src/ScintillaComponent/FindReplaceDlg.cpp

```diff
@@ -5631,14 +5649,33 @@ int Progress::createProgressWindow()
 	int yTextPos = dpiManager.scaleY(5);
 	auto ctrlWidth = width - widthPadding - xStartPos;
 
-	_hPText = ::CreateWindowEx(0, TEXT("STATIC"), TEXT(""),
+	_hPathText = ::CreateWindowEx(0, TEXT("STATIC"), TEXT(""),
 		WS_CHILD | WS_VISIBLE | SS_PATHELLIPSIS,
 		xStartPos, yTextPos,
 		ctrlWidth, textHeight, _hwnd, NULL, _hInst, NULL);
 
+	NativeLangSpeaker* pNativeSpeaker = (NppParameters::getInstance()).getNativeLangSpeaker();
+	generic_string hits = pNativeSpeaker->getLocalizedStrFromID("progress-hits-title", TEXT("Hits:"));
+	_hRunningHitsStaticText = ::CreateWindowEx(0, TEXT("STATIC"), hits.c_str(),
+		WS_CHILD | WS_VISIBLE | SS_RIGHT,
+		xStartPos, yTextPos + textHeight * 2,
+		75, textHeight, 
+		_hwnd, NULL, _hInst, NULL);
+
+	_hRunningHitsText = ::CreateWindowEx(0, TEXT("STATIC"), TEXT(""),
+		WS_CHILD | WS_VISIBLE,
+		xStartPos + 75 + 2, yTextPos + textHeight * 2,
+		70, textHeight, 
+		_hwnd, NULL, _hInst, NULL);
+
 	HFONT hf = (HFONT)::GetStockObject(DEFAULT_GUI_FONT);
 	if (hf)
-		::SendMessage(_hPText, WM_SETFONT, reinterpret_cast<WPARAM>(hf), MAKELPARAM(TRUE, 0));
+	{
+		for (auto const& item : { _hPathText, _hRunningHitsStaticText, _hRunningHitsText })
+		{
+			::SendMessage(item, WM_SETFONT, reinterpret_cast<WPARAM>(hf), MAKELPARAM(TRUE, 0));
+		}

```

Comment:
Why do we need a loop here, which takes 4 lines, whereas we can write only 3 lines explicitly?

Reply:
In the unrolling of the loop I noticed an additional efficiency I could take by combining the set-font calls for 4 controls (instead of 3) together under one `if (hf)`, so I did that as well.